archey/distributions.py

```diff
@@ -37,6 +37,7 @@ class Distributions(Enum):
     LINUX_MINT = 'linuxmint'
     OPENBSD = 'openbsd'
     OPENSUSE = 'opensuse'
+    POP = 'pop'
```

Comment:
Hey !
One question here too : do you know when Pop! OS decided to ship its own `pop` ID as we can see (here)(https://github.com/KittyKatt/screenFetch/issues/675#issuecomment-632126301) ?
One more proposal : should we provide a "manual" detection procedure for previous (?) Pop! OS releases which were identified as Ubuntu (as your screenshot tells) ?

Thanks again.
Bye 👋

Reply:
I am unsure which version the id was added, but I can try to locate the iso to test previous 18.* & 19.* versions (no longer supported by System76). Perhaps, `cat /etc/*-release` on older releases gives a similar out pictured below:
![cat-etc-release](https://user-images.githubusercontent.com/53982860/95023272-0317fd00-0631-11eb-8598-7eb4549b32e2.png)
And if the `pop` id is unavailable on older releases, then a workaround could be parsing the `PRETTY_NAME` attribute for `Pop*`.

Additionally, I added an alt Pop!_OS logo by the name `popos`, and fixed a minor color mistake on the other logo. Plan to merge updates soon.
![archey-local-alt](https://user-images.githubusercontent.com/53982860/95022829-58064400-062e-11eb-83f8-a8950826fe73.png)

Reply:
Thanks for your quick responses. See my points below :

1. I'm okay about implementing a special case for this distribution, expect a commit here soon. Would you be able to try it out ?

2. About alternative logos, unless Neofetch for instance, Archey does not support this mechanism to keep things simple. I'd preferred you only propose the first (fixed) one here. Distribution identifiers should be unique (and match `os-release` if possible). So I guess we should keep `pop` as our candidate.

Bye, 👋

Reply:
Oh sorry I've just understood the screenshot you added in the OP had been taken on your Ubuntu.
No need for a special detection case then (unless previous releases of this OS didn't ship `pop` as `ID` ? Will you be able to easily check this ? I couldn't find Pop's base-files/skeleton/rootfs to verify myself...).
From my PoV, we're good here once your logo fix lands there.

Bye, sorry again for the noises 🙇

Reply:
Logo color changes have been made. All good to go for **Pop!_OS 20.04 LTS**. And okay, I did not include the alternate logo.

![archey-local-final](https://user-images.githubusercontent.com/53982860/95028487-57cd6f00-0655-11eb-9d7c-c8e84481cd7e.png)

However, checked the **18.04** and **19.10** versions using `cat /etc/*-release` command and they do not include a unique `pop` id, so a special detection case will be needed.

**18.04**
![cat-etc-pop18](https://user-images.githubusercontent.com/53982860/95028409-f0172400-0654-11eb-939b-b1de3763e46f.png)

**19.10**
![cat-etc-pop19](https://user-images.githubusercontent.com/53982860/95028411-f4434180-0654-11eb-8b33-5130345338d6.png)

Reply:
What a coincidence then 😂
See my patch below, I'm waiting for your feedback before merging this.

Thanks for your logo fix 👌
Bye 👋